<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>386</width>
    <height>467</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="MyGLWidget" name="myGLWidget" native="true"/>
    </item>
    <item>
     <widget class="QSlider" name="xRotSlider">
      <property name="maximum">
       <number>360</number>
      </property>
      <property name="singleStep">
       <number>16</number>
      </property>
      <property name="pageStep">
       <number>15</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="tickPosition">
       <enum>QSlider::TicksAbove</enum>
      </property>
      <property name="tickInterval">
       <number>15</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSlider" name="yRotSlider">
      <property name="maximum">
       <number>360</number>
      </property>
      <property name="singleStep">
       <number>16</number>
      </property>
      <property name="pageStep">
       <number>15</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="tickPosition">
       <enum>QSlider::TicksAbove</enum>
      </property>
      <property name="tickInterval">
       <number>15</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSlider" name="zRotSlider">
      <property name="maximum">
       <number>360</number>
      </property>
      <property name="singleStep">
       <number>16</number>
      </property>
      <property name="pageStep">
       <number>15</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="tickPosition">
       <enum>QSlider::TicksAbove</enum>
      </property>
      <property name="tickInterval">
       <number>15</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>386</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>MyGLWidget</class>
   <extends>QWidget</extends>
   <header>myglwidget.h</header>
   <container>1</container>
   <slots>
    <slot>setXRotation(int)</slot>
    <slot>setYRotation(int)</slot>
    <slot>setZRotation(int)</slot>
   </slots>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>xRotSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>myGLWidget</receiver>
   <slot>setXRotation(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>29</x>
     <y>337</y>
    </hint>
    <hint type="destinationlabel">
     <x>44</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>yRotSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>myGLWidget</receiver>
   <slot>setYRotation(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>154</x>
     <y>374</y>
    </hint>
    <hint type="destinationlabel">
     <x>136</x>
     <y>265</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>zRotSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>myGLWidget</receiver>
   <slot>setZRotation(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>213</x>
     <y>423</y>
    </hint>
    <hint type="destinationlabel">
     <x>214</x>
     <y>281</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
